rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -56,13 +63,15 @@ def to_string(self, ts_converter: Callable[[Timestamp], str]) -> str:
     def to_exported_dict(
             self,
             ts_converter: Callable[[Timestamp], str],
-            eth_explorer: Optional[str],
-            for_api: bool,
+            export_type: AccountingEventExportType,
```

Comment:
Hmm but this still does not show the connection I meant needs to be typechecked. Please add two typing overloads here showing that if the export_type == AccountingEventExportType.CSV then the evm_explorer needs to be given.